src/js/extension.js

```diff
@@ -0,0 +1,73 @@
+var Extension;
+(function(){
+
+    /* global Util */
+
+    Extension = function (options) {
+        Util.extend(this, options);
```

Comment:
(take Anchor for example. some of the form label options were defined on the .prototype of the anchor extension, and the rest of the options are read from the core options hash + defaults. for consistency one would either move those prototype props into a default "options" object, and migrate the core options to that, or "just migrate core options to the extension prototype". the latter way means all the defaults and definitions are defined right there at the extension in a single place, and "justworks" for preserving defaults through prototypical inheritance rather than carrying around an object literal you have to work to mix and overwrite (.options) ... maybe it's a personal preference, but I see no real distinction between protoprops and an explicit opions object, and would go so far as to say the options object path makes things more complicated.